assets/dev/js/editor/utils/container-helper.js

```diff
@@ -70,125 +70,129 @@ export class ContainerHelper {
 	 * Create a Container element based on a preset.
 	 *
 	 * @param {string} preset
-	 * @param {Container} [container=elementor.getPreviewContainer()]
+	 * @param {Container} [target=elementor.getPreviewContainer()]
 	 * @param {Object} [options={}]
```

Comment:
Add `createWrapper` and other known options

Reply:
Handled.